<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ayat-qt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter_1">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="selectSura_spinBox">
               <property name="maximumSize">
                <size>
                 <width>50</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>114</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="selectSura_lineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>110</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="suwarList_tableWidget">
             <property name="maximumSize">
              <size>
               <width>165</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <attribute name="horizontalHeaderVisible">
              <bool>false</bool>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QSpinBox" name="selectAya_spinBox">
               <property name="minimum">
                <number>1</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="selectAya_lineEdit"/>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>: Search in Sura</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="ayatList_listWidget">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="wordWrap">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <layout class="QVBoxLayout" name="search_layout">
         <property name="spacing">
          <number>1</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="searchTool_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="searchTool_label">
             <property name="text">
              <string>: Search in Quran</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="searchTool_tableWidget"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="tafsir_layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="tafasirNames_comboBox"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="tafsir_textBrowser">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="html">
           <string notr="true"></string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="actions">
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="copySelectedAct"/>
    <addaction name="exportSearchAct"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="language">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="langArabicAct"/>
    <addaction name="langEnglishAct"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Recitation</string>
    </property>
    <widget class="QMenu" name="recitationsMenu">
     <property name="title">
      <string>Select Reciter</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="readAyatOnClickAct"/>
    <addaction name="recitationsMenu"/>
    <addaction name="separator"/>
    <addaction name="autoDownloadAct"/>
    <addaction name="downloadAudioAct"/>
   </widget>
   <widget class="QMenu" name="tools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="showSearchAct"/>
    <addaction name="showTafsirAct"/>
   </widget>
   <addaction name="actions"/>
   <addaction name="menu"/>
   <addaction name="tools"/>
   <addaction name="language"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exitAct">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="copySelectedAct">
   <property name="text">
    <string>Copy Selected Ayat</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+C</string>
   </property>
  </action>
  <action name="langArabicAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true">عربية</string>
   </property>
  </action>
  <action name="langEnglishAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string notr="true">English</string>
   </property>
  </action>
  <action name="readAyatOnClickAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Recite Ayat On Click</string>
   </property>
  </action>
  <action name="downloadAudioAct">
   <property name="text">
    <string>Download Recitations</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+D</string>
   </property>
  </action>
  <action name="showSearchAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Search Tool</string>
   </property>
  </action>
  <action name="showTafsirAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Tafasir Tool</string>
   </property>
  </action>
  <action name="exportSearchAct">
   <property name="text">
    <string>Export Search</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+S</string>
   </property>
  </action>
  <action name="autoDownloadAct">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto Download</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
